src/Async/Qu.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  Qu.swift
+//
+//  Created by Logan Wright on 10/24/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+public typealias Block = () -> ()
```

Comment:
nope. i think at this point we dont really know what the real problems are. you defiitely have a better understanding of this than me. I think we should take this as a note to ourselves, and at the first point where we see there is a problem we should get this modified.